### PR TITLE
#MAN-202 Remove space between entities

### PR DESCRIPTION
**Pull request by cdoyle-temple**
#MAN-202 
Similar to what we did on the main homepage, can you remove the space between the building entities on the index page and instead have more white space on the outside of the three columns.

*********************
- Changed rows of three to rows of four in order to minimize space betw…een images when window stretches.
- Also optimized calls to and display of images for index and show pages.

### CODE REVIEW

File: app/models/building.rb [hunk 1]
```diff
@@ -20,6 +21,17 @@ class Building < ApplicationRecord
   before_validation :normalize_phone_number
   before_validation :sanitize_description
 
+  def index_image
+    variation =
+      ActiveStorage::Variation.new(Uploads.resize_to_fill(width: 250, height: 190, blob: photo.blob, gravity: "Center"))
```
Comment (nomadicoder) [inline app/models/building.rb]: Should we parameterize the width and height somewhere?

File: app/models/building.rb [hunk 2]
```diff
@@ -20,6 +21,17 @@ class Building < ApplicationRecord
...
+    ActiveStorage::Variant.new(photo.blob, variation)
+  end
+  def show_image
+    variation =
+      ActiveStorage::Variation.new(Uploads.resize_to_fill(width: 600, height: 450, blob: photo.blob, gravity: "Center"))
```
Comment (nomadicoder) [inline app/models/building.rb]: Parameterize literal values?

File: app/assets/stylesheets/buildings.scss [hunk 3]
```diff
@@ -22,7 +22,8 @@ div.required  label::after {
 		margin: auto;
 		background-color: $background-grey;
 		max-width:680px;
-		height:450px;
+		height: auto;
+		min-height:450px;
```
Comment (nomadicoder) [inline app/assets/stylesheets/buildings.scss]: Parameterize 450px somewhere?

File: app/assets/stylesheets/buildings.scss [hunk 4]
```diff
@@ -77,7 +78,14 @@ div.required  label::after {
     overflow: hidden;
     padding: 0;
     width: 250px;
-    height: 190px;
+    height: 189px;
```
Comment (nomadicoder) [inline app/assets/stylesheets/buildings.scss]: Parameterize 189px somewhere?

File: app/models/building.rb (hunk 1 above)
Comment (cdoyle-temple) [reply app/models/building.rb]: It's currently this way all over the site. That would be a bigger ticket than this. Can we do it there (in the new ticket) if necessary?

File: app/assets/stylesheets/buildings.scss (hunk 3 above)
Comment (cdoyle-temple) [reply app/assets/stylesheets/buildings.scss]: This needs to be this way in the css.

File: app/assets/stylesheets/buildings.scss (hunk 4 above)
Comment (cdoyle-temple) [reply app/assets/stylesheets/buildings.scss]: This needs to be this way in the css.

File: app/models/building.rb (hunk 2 above)
Comment (cdoyle-temple) [reply app/models/building.rb]: See above comment.

File: app/models/building.rb (hunk 1 above)
Comment (nomadicoder) [reply app/models/building.rb]: Thanks. I'll resolve this.

**Pull request merged**